### DISCUSSION TITLE
Actions jobs for this repository have been disabled

### DISCUSSION THREAD

**Discussion by laszlo-san** [community/community#17493] [Actions]
Hey All!

I am an owner of an Org and want to run Actions on one of our repos. I constantly get failed with the information:
`Actions jobs for this repository have been disabled by GitHub staff. If you are the repository owner, you can contact support via https://github.com/contact for more information.`

Already created a Ticket for this a week ago, but the Support didn't bat an eye on that.

The problem is not with just that one Org repo, created several test repos and got the same result. I tried the same action, or even more complicated ones in my personal repos, and those just worked. I did check persmissions, Org level Action execution permissions, repo level Action execution permissions without any success.

What may happening, what am I missing?

**Comment by fedekunze** [comment]
@laszlo-san were you able to resolve the issue? We're having the same problem: https://github.com/github-community/community/discussions/19015#discussioncomment-3016524
